requirements.txt

```diff
@@ -8,6 +8,7 @@ Pillow>=7.1.2
 PyYAML>=5.3.1
 requests>=2.23.0
 scipy>=1.4.1
+torchvision>=0.8.1
```

Comment:
Is this change related? Otherwise lgtm

Reply:
Whoops that's unrelated. Thanks for catching